template_handler/handler.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from sceptre.template_handlers import TemplateHandler
+
+
+class CustomTemplateHandler(TemplateHandler):
+    """
+    The following instance attributes are inherited from the parent class TemplateHandler.
+
+    Parameters
+    ----------
+    name: str
+        The name of the template. Corresponds to the name of the Stack this template belongs to.
+    handler_config: dict
+        Configuration of the template handler. All properties except for `type` are available.
+    sceptre_user_data: dict
+        Sceptre user data defined in the Stack config
+    connection_manager: sceptre.connection_manager.ConnectionManager
+        Connection manager that can be used to call AWS APIs
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(CustomTemplateHandler, self).__init__(*args, **kwargs)
```

Comment:
This really isn't necessary. You could remove this bit.

Reply:
it's in the docs so i'll just leave as-is unless you feel strongly about removing it.